paywall/src/paywall-script/index.ts

```diff
@@ -47,6 +52,8 @@ export class Paywall {
 
   lockStatus?: string
 
+  provider?: Enabler
```

Comment:
We have to decide how to signal the presence or absence of this provider to the checkout. We can intercept the setconfig call and include the presence of the provider as a value in the config that would be for internal use only which would limit the number of things we have to wait for

Reply:
yes, I think we can add it to the config.